src/components/shop-cart-item.js

```diff
@@ -162,62 +168,55 @@ $_documentContainer.innerHTML = `<dom-module id="shop-cart-item">
 
     </style>
 
-    <a href\$="/detail/[[entry.item.category]]/[[entry.item.name]]" title\$="[[entry.item.title]]">
```

Comment:
Let's remove this one last `@apply` usage: https://github.com/Polymer/shop/blob/2d10d3b8f6a6e1a7fb903befd4a5fa5467702e3c/src/components/shop-cart-item.js#L148-L150